code/controllers/subsystem/ast.dm

```diff
@@ -0,0 +1,17 @@
+SUBSYSTEM_DEF(ast)
```

Comment:
![image](https://user-images.githubusercontent.com/1234602/27774053-081bb492-5f57-11e7-8a2e-8d102e8a43b1.png)

Doesn't matter I guess, just one more thing to rewrite.